source/extensions/network/dns_resolver/cares/dns_impl.cc

```diff
@@ -228,8 +243,10 @@ void DnsResolverImpl::AddrInfoPendingResolution::onAresGetAddrInfoCallback(
     // that the first lookup failed to return any addresses. Note that DnsLookupFamily::All issues
     // both lookups concurrently so there is no need to fire a second lookup here.
     if (dns_lookup_family_ == DnsLookupFamily::Auto) {
+      family_ = AF_INET;
```

Comment:
Why is this change and the one below needed?

Reply:
They are not needed, I just found it's a bit confusing while debugging it since the lookup family_ has changed for the second request, happy to revert this.

Reply:
OK got it, makes sense. Fine to keep.